openapi/listings/listings.yaml

```diff
@@ -447,3 +571,316 @@ components:
         The listing score information for a business location
       x-tags:
         - Listing Scores
+    listingProfiles:
+      type: object
+      x-examples:
+        Example 1:
+          id: AG-1234567
+          type: businessLocations
+          attributes:
+            customerIdentifier: string
+            name: string
+            address:
+              line1: 109 8th Street E.
+              line2: Suite 23
+              city: string
+              stateCode: CA
+              zip: string
+              postalCode: S7M 1R3
+              regionCode: CA-SK
+              countryCode: CA
+            phoneNumbers:
+              - string
+            serviceAreaBusiness: true
+            geoCoordinate:
+              latitude: -90
+              longitude: -180
+            hours:
+              - hoursTypeId: general
+                regularHours:
+                  - openDay: wednesday
+                    closeDay: wednesday
+                    openTime: '09:45'
+                    closeTime: '09:45'
+                specialHours:
+                  - status: closed
+                    startDate: '2019-08-24'
+                    endDate: '2019-08-24'
+                    openTime: '09:45'
+                    closeTime: '09:45'
+            updatedAt: string
+          relationships:
+            businessPartner:
+              data:
+                type: partners
+                id: ABC
+            businessCategories:
+              data:
+                - type: businessCategories
+                  id: 'active:diving:freediving'
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        A listing profile is the representation of a business location (or similar) used by the Listing Builder product. It contains advanced fields used by listing sources. If the Listing Builder product is active for the location the data will be shared to listing sources such as Google Maps.
+      properties:
+        id:
+          type: string
+          example: AG-1231232
+          description: An identifier for the record. Currently these will be the same as a platform/salesAccounts or platform/businessLocations id.
+        type:
+          type: string
+          enum:
+            - listingProfiles
+          default: listingProfiles
+          description: The data type of this record
+        attributes:
+          type: object
+          readOnly: true
```

Comment:
By marking the attribute object as readonly none of the fields under it display as updatable in the docs.

![image](https://github.com/vendasta/api-gateway-docs/assets/6698651/6e587019-acf3-4e0d-8c00-61628b180cd3)


```suggestion
        attributes:
          type: object
```